orchagent/intfsorch.cpp

```diff
@@ -1015,3 +1087,30 @@ void IntfsOrch::doTask(SelectableTimer &timer)
         }
     }
 }
+
+void IntfsOrch::getNatSupportedInfo()
```

Comment:
I see that you are repeating this same function in multiple places. Can you do this query one time, like in switchorch, and use a single API to check if NAT is supported.

Reply:
Ok. Moved this functionality to main.cpp file and updating the global variable 'gIsNatSupported'.